### PR TITLE
Issue#10

### PR DESCRIPTION
**Pull request by ArnautS**
Added the force command + some more cleanup

### CODE REVIEW

File: cogs/announcements.py [hunk 1]
```diff
@@ -99,61 +99,60 @@ def __init__(self, bot, utils):
         self.utils.save_settings()
         await context.message.add_reaction('\U0001F44D')
 
+    @commands.command(name='force')
```
Comment (th8) [inline cogs/announcements.py]: This command can be included in the @announcement category like all other commands.

**Comment by ghost** [comment]
I more or less meant something that retrieved entries from reddit itself, as it's done here:
https://github.com/RMTweedeKamer/Griffier/blob/006b700e8e8578954c05356a87b3453cf96d746c/cogs/announcements.py#L115-L117

**Review by th8 — CHANGES_REQUESTED** [review]

File: cogs/announcements.py (hunk 1 above)
Comment (ArnautS) [reply cogs/announcements.py]: Is it still commands.has_any_role on the next line?

Comment (th8) [reply cogs/announcements.py]: Yep

**Pull request merged**